plugins/org/src/plugin.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createPlugin, createRouteRef } from '@backstage/core';
+
+export const rootRouteRef = createRouteRef({
+  path: '',
+  title: 'Groups',
+});
+
+export const viewGroupRouteRef = createRouteRef({
+  path: ':groupName',
+  title: 'View Group',
+});
+
+export const viewMemberRouteRef = createRouteRef({
+  path: 'member/:memberName',
+  title: 'View Member',
+});
```

Comment:
Are these routes still required? I tried the PR out but could only find the entity routes

Reply:
You are right, those routes are unused now, I forgot to remove them when doing cleanup. Thanks 👍